frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -64,69 +61,67 @@ export const NavigationSidebar = () => {
 
     return (
         <StretchContainer>
-            <ScreenHeightBox>
-                <PrimaryNavigationList
+            <PrimaryNavigationList
+                mode={mode}
+                onClick={setActiveItem}
+                activeItem={activeItem}
+            />
+            <SecondaryNavigation
+                expanded={expanded.includes('configure')}
+                onExpandChange={(expand) => {
+                    changeExpanded('configure', expand);
+                }}
```

Comment:
The logic here could be encapsulated inside SecondaryNavigation, but that would require adding more properties so maybe not an actual improvement :)